velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -99,6 +100,18 @@ void readData(ReadFile* readFile) {
   ASSERT_EQ(warfFromBuf, "abbbbbcc");
 }
 
+void writeData(HdfsWriteFile* writeFile, const char* data) {
+  std::string data_str(data);
+  ASSERT_EQ(data_str, "abcdefghijk");
```

Comment:
It is strange to assert on input values. If this method supports only one input, then should it just remove the 'data' parameter?